command/certificate/format.go

```diff
@@ -51,12 +64,72 @@ Convert PEM format to DER and write to disk:
 '''
 $ step certificate format foo.pem --out foo.der
 '''
+
+Convert a .p12 file to a certificate and private key:
+
+'''
+$ step certificate format foo.p12 --crt foo.crt --key foo.key --format pem
+'''
+
+Convert a .p12 file to a certificate, private key and intermediate certificates:
+
+'''
+$ step certificate format foo.p12 --crt foo.crt --key foo.key --ca intermediate.crt --format pem
+'''
+
+Convert a certificate and private key to a .p12 file:
+
+'''
+$ step certificate format foo.crt --crt foo.p12 --key foo.key --format p12
+'''
+
+Convert a certificate, a private key, and intermediate certificates(s) to a .p12 file:
+
+'''
+$ step certificate format foo.crt --crt foo.p12 --key foo.key \
+  --ca intermediate-1.crt --ca intermediate-2 --format p12
+'''
```

Comment:
Perhaps the first argument should not be mandatory because right now `--crt` is used both as input and as ouput.

These should be valid commands:
```sh
# To stdout:
step certificate format foo.pem 
step certificate format --format p12 --crt ... --key ... --ca ...
# To a file:
step certificate format foo.pem -out foo.der
step certificate format --format p12 --crt f.pem --key f.key --ca ca.pem --out foo.p12
# To multiple files:
step certificate format foo.p12 --crt f.pem --key f.key --ca ca.pem
```

Reply:
I don't think the main argument was mandatory before. But I think my update makes that more clear.